Bottom: de7811bd490773e00fd6656717cbf3f10c542bde
Top:    e8bea7e5a050172cb24cb267d7872f0c38173f45
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-27 02:04:31 +0100

Refresh of add-soldermask-layers

---

diff --git a/src/global.h b/src/global.h
index 1d017fe..bcbbe29 100644
--- a/src/global.h
+++ b/src/global.h
@@ -195,7 +195,7 @@ OutputType;
 typedef struct
 {
   Cardinal Number[MAX_LAYER],	/* number of entries per groups */
-    Entries[MAX_LAYER][MAX_LAYER + 2];
+    Entries[MAX_LAYER][MAX_LAYER + 4];
 } LayerGroupType;
 
 struct BoxType		/* a bounding box */
@@ -397,7 +397,7 @@ typedef struct			/* holds all objects */
   rtree_t *via_tree, *element_tree, *pin_tree, *pad_tree, *name_tree[3],	/* for element names */
    *rat_tree;
   struct PCBType *pcb;
-  LayerType Layer[MAX_LAYER + 2];	/* add 2 silkscreen layers */
+  LayerType Layer[MAX_LAYER + EXTRA_LAYERS];	/* add 2 silkscreen layers and 2 soldermask layers */
   int polyClip;
 } DataType;
